inc/composer/class-command.php

```diff
@@ -320,18 +335,130 @@ protected function logs( InputInterface $input, OutputInterface $output ) {
 	protected function shell( InputInterface $input, OutputInterface $output ) {
 		$columns = exec( 'tput cols' );
 		$lines = exec( 'tput lines' );
+		$command_prefix = $this->get_base_command_prefix();
 		passthru( sprintf(
-			'cd %s; VOLUME=%s COMPOSE_PROJECT_NAME=%s docker-compose exec -e COLUMNS=%d -e LINES=%d php /bin/bash',
-			'vendor/altis/local-server/docker',
-			escapeshellarg( getcwd() ),
-			$this->get_project_subdomain(),
+			"$command_prefix docker-compose exec -e COLUMNS=%d -e LINES=%d php /bin/bash",
 			$columns,
 			$lines
 		), $return_val );
 
 		return $return_val;
 	}
 
+	protected function db( InputInterface $input, OutputInterface $output ) {
+		$db = $input->getArgument( 'options' )[0] ?? '';
```

Comment:
```suggestion
		$db = $input->getArgument( 'options' )[0] ?? null;
```
Just going to test this out, I think it's necessary for the default command to run.

Reply:
I've tested both ways and it always works but anyway, `null` looks slightly better

Reply:
Ahh of course, the `switch` doesn't do strict matching. It's a `==` comparison 👍